### ISSUE TITLE
Chips should be `<button>`s

### ISSUE THREAD

**Issue by coliff** [trimble-oss/modus-web-components#2254]
Chips are buttons. They are focusable and can be clicked/tapped. We should change them from a `<div>` to a `<button>`. Should be relatively straightforward with minimal CSS changes needed.
